kod/object/active/holder/nomoveon/battler/player/user/dm/admin.kod

```diff
@@ -159,6 +161,17 @@ messages:
             if Send(i,@DMCreateItemAtt,#who=self,#string=string,
                     #oWeapon=oWeapon)
             {
+               % Identify the newly-added attribute
+               if (NOT StringContain(string, admin_create_unrevealed_itematt_command))
```

Comment:
It feels weird to check the user's speech again here.  How about setting a boolean flag earlier, saying whether the attribute should be revealed?  That will also be less fragile in the future.

Reply:
Done.